### PR TITLE
Removes synaptizine from sensory restoration

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
> symptom to fix your eyes
> it's actually infinite adrenals

???

### CODE REVIEW

**Comment by KazeEspada** [comment]
Thats what you get for merging @LanCartwright 's garbage.

**Comment by kevinz000** [comment]
Goddamn meme lead. >:v

**Comment by Fox-McCloud** [comment]
:-1: 

It's about the only thing that makes this symptom even vaguely worth taking, considering all the other things this fixes are hardly ever a thing in the day to day lives of spessmen.

**Comment by KorPhaeron** [comment]
I mean I guess the symptom could just be axed then

**Comment by LanCartwright** [comment]
Remind me tonight. I'll rework it.

**Comment by KorPhaeron** [comment]
Sure

File: code/datums/diseases/advance/symptoms/sensory.dm [hunk 1]
```diff
@@ -38,12 +38,12 @@ Bonus
...
 			if(4)
-				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10 && M.reagents.get_reagent_amount("synaptizine") < 10)
-					M.reagents.add_reagent_list(list("antihol"=10, "inacusiate"=10, "synaptizine"=5))
+				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10)
+					M.reagents.add_reagent_list(list("antihol"=10, "inacusiate"=10))
 					M << "<span class='notice'>You feel focused.</span>"
 			if(5)
-				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10 && M.reagents.get_reagent_amount("synaptizine") < 10 && M.reagents.get_reagent_amount("mannitol") < 10)
-					M.reagents.add_reagent_list(list("mannitol"=10, "antihol"=10, "inacusiate"=10, "synaptizine"=10))
+				if(M.reagents.get_reagent_amount("antihol") < 10 && M.reagents.get_reagent_amount("inacusiate") < 10 && M.reagents.get_reagent_amount("mannitol") < 10)
```
Comment (phil235) [inline code/datums/diseases/advance/symptoms/sensory.dm]: Oh GOD....

Comment (Dorsisdwarf) [inline code/datums/diseases/advance/symptoms/sensory.dm]: oh god?

Comment (phil235) [inline code/datums/diseases/advance/symptoms/sensory.dm]: the code is hacky :disappointed:

**Comment by KorPhaeron** [comment]
Okay allegedly J_Mad is fixing this we'll see

**Pull request closed without merging**